apps/settings/js/icc.js

```diff
@@ -500,12 +526,19 @@
       li.appendChild(small);
     }
 
+    if (entry.icon && STKHelper) {
```

Comment:
I think we should use requirejs to help us require `STKHelper` here. By doing so, you don't have to keep checking whether STKHelper exists or not.